packages/core/src/utils/nuxt/_proxyUtils.ts

```diff
@@ -43,12 +43,14 @@ export const getIntegrationConfig = (context: NuxtContext, configuration: any) =
     baseURL = `/${baseURL}`;
   }
 
+  const resolvedHost = context?.req?.headers?.['x-forwarded-host'] || context?.req?.headers?.host;
+
   return merge({
     axios: {
       baseURL,
       headers: {
         ...(cookie ? { cookie } : {}),
-        ...(context.req ? { Host: context.req.headers.host } : {})
+        ...(resolvedHost ? { Host: resolvedHost, 'x-forwarded-host': resolvedHost } : {})
```

Comment:
We don't need to add `'x-forwarded-host'` in here, cloud infra will do it on their side anyway. We should be just aware that when a traffic comes to the app from outside it comes as x-forwarded-host

Reply:
Removed thanks for clarifying that.